canonical-websites/www.ubuntu.com
# 1601 containers overview

## Done

* update /containers for vbt 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/containers)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/4.%20Containers?preview=containers-overview.png) and the [demo](http://www.ubuntu.com-1601-containers-overview.demo.haus/containers)

## Issue / Card

Fixes #1601
